lib/airrecord.rb

```diff
@@ -9,4 +9,10 @@ module Airrecord
   extend self
   Error = Class.new(StandardError)
   attr_accessor :api_key
+  attr_accessor :throttle
```

Comment:
Is this just so that users can choose to disable rate-limiting?

Reply:
Si. Someone might be on a premium plan and have negotiated higher limits.